lms/templates/emails/business_order_confirmation_email.txt

```diff
@@ -1,5 +1,8 @@
 ## mako
-<%! from django.utils.translation import ugettext as _ %>
+<%page expression_filter="h"/>
```

Comment:
Note: this is an odd example of an email template that contains HTML tags.  So, it should probably be HTML-escaped, but should move to a separate PR as described elsewhere to possibly do further review.  Why is the file named `.txt`?